src/ol/expr/cpu.js

```diff
@@ -20,30 +20,37 @@ import {ColorType, LiteralExpression, Ops, parse} from './expression.js';
  * value.  The evaluator function should do as little allocation and work as possible.
  */
 
+export const UNKNOWN_VALUE = {};
```

Comment:
Minor nit, I think `UNKNOWN` would sufficiently communicate what this is about (no need for `_VALUE`).